Packs/Forcepoint_Security_Management_Center/Integrations/Forcepoint_Security_Management_Center/Forcepoint_Security_Management_Center.py

```diff
@@ -0,0 +1,240 @@
+
+    outputs = {'Name': ip_list.name,
+               'Addresses': ip_list.iplist,
+               'Comment': ip_list.comment}
+
+    return CommandResults(
+        outputs_prefix='ForcepointSMC.Address',
+        outputs=outputs,
+        raw_response=outputs,
+        outputs_key_field='Name',
+        readable_output=f'IP List {name} was updated successfully.'
+    )
+
+
+def list_address_command(args: Dict[str, Any]) -> CommandResults:
+    """Lists the IP Lists in the system.
+
+    Args:
+        args (Dict[str, Any]): The command args.
+
+    Returns:
+        CommandResults
+    """
+    name = args.get('name', '')
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    all_results = argToBoolean(args.get('all_results', False))
+
+    ip_lists = []
+    if name:
+        ip_lists = list(IPList.objects.filter(name))
+    elif all_results:
+        ip_lists = list(IPList.objects.all())
+    else:
+        ip_lists = list(IPList.objects.limit(limit))
+
+    outputs = []
+    for ip_list in ip_lists:
+        outputs.append({'Name': ip_list.name,
+                        'Addresses': ip_list.iplist,
+                        'Comment': ip_list.comment})
+
+    return CommandResults(
+        outputs_prefix='ForcepointSMC.Address',
+        outputs=outputs,
+        raw_response=outputs,
+        outputs_key_field='Name',
+        readable_output=tableToMarkdown(name='IP Lists:', t=outputs, removeNull=True),
+    )
+
+
+def delete_address_command(args: Dict[str, Any]) -> CommandResults:
+    """Deleting IP List with a list of addresses.
+
+    Args:
+        args (Dict[str, Any]): The command args.
+
+    Returns:
+        CommandResults
+    """
+    name = args.get('name')
+
+    try:
+        ip_list = IPList(name).delete()
+
+    except ElementNotFound:
+        return CommandResults(readable_output=f'IP List {name} was not found.')
+
+    outputs = {'Name': ip_list.name,
+               'Deleted': True}
+
+    return CommandResults(
+        outputs_prefix='ForcepointSMC.Address',
+        outputs_key_field='Name',
+        outputs=outputs,
+        readable_output=f'IP List {name} was deleted successfully.'
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+    """
+    params = demisto.params()
+    url = params.get('url')
+    api_key = params.get('credentials', {}).get('password')
+    port = params.get('port')
+    insecure = params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    command = demisto.command()
+
+    demisto.debug(f'Command being called is {command}')
+    try:
+
+        client = Client(
+            url=url,
+            api_key=api_key,
+            insecure=insecure,
+            proxy=proxy,
+            port=port)
+
+        if demisto.command() == 'test-module':
+            result = test_module(client)
```

Comment:
let's do the login in the client c'tor and in the test module run a simple method like `IPList.objects.limit(1)`